examples/image_expansion.py

```diff
@@ -0,0 +1,174 @@
+"""
+This script illustrates several expansion methods for 2D images developed in ASPIRE package
+based on the basis functions of  Fourier Bessel (FB) and Prolate Spheroidal Wave Function (PSWF).
+"""
+
+import os
+import logging
+import numpy as np
+import timeit
+
+from aspire.basis.fb_2d import FBBasis2D
+from aspire.basis.ffb_2d import FFBBasis2D
+from aspire.basis.pswf_2d import PSWFBasis2D
+from aspire.basis.fpswf_2d import FPSWFBasis2D
+from aspire.utils.matrix import anorm
+import matplotlib.pyplot as plt
+
+
+logger = logging.getLogger('aspire')
+
+
+logger.info('This script illustrates different image expansion methods in ASPIRE package.')
+
+# Load the images from numpy array, 10 images of Ribosome 70S with size of 129 X 129
```

Comment:
“numpy” → “NumPy”. “Ribosome 70S” → “70S ribosome”. Remove second “of”.

Reply:
Done.